src/main/java/class3_fileReaderWriter/eveningPractice/SeventhTask.java

```diff
@@ -0,0 +1,70 @@
+package class3_fileReaderWriter.eveningPractice;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SeventhTask {
+    private static final String SCV_FILE = "users.csv";
+    private static final String SCV_SEPARATOR = ", ";
+
+    public static void main(String[] args) {
+        System.out.println("******");
+
+        System.out.println("");
+        removeUserInfo("Lena");
+        System.out.println("");
+        getUsersFromFile();
+    }
+
+    public static void removeUserInfo(String name) {
+        List<User> users = getUsersFromFile();
+        List<User> updatedUsers = users.stream()
+                .filter(user -> !user.getName().equalsIgnoreCase(name))
+                .toList();
```

Comment:
again the usage of streams here, as convenient as it might be, I'd prefer to see a simpler solution.